webdriver-spec.html

```diff
@@ -3802,7 +3802,7 @@
 <ol>
  <li><p>Let <var>rectangle</var> be
   the first element of the <a><code>DOMRect</code></a> sequence
-  returned by calling <a>getClientRects</a> on <var><a>element</a></var>.
+  returned by calling <a>getClientRects</a> on the <a>element</a>.
```

Comment:
I _think_ I resolved this properly, let me know and sorry for the back and forth.